browser/BUILD.gn

```diff
@@ -1,5 +1,5 @@
 import("//brave/build/config.gni")
-import("//brave/components/speedreader/buildflags.gni")
```

Comment:
Can't we just drop (vs fix) this include as well? I don't see it used anywhere in this file either...